examples/sanity/app/lib/sanity/usePreviewSubscription.ts

```diff
@@ -27,29 +30,36 @@ export function usePreviewSubscription(query, subscriptionOptions) {
         documentLimit: 1000,
       });
 
-      store.subscribe(
+      sub = store.subscribe<unknown>(
         query,
         params ?? {}, // Params
         (err, result) => {
           if (err) {
             console.error("Oh no, an error:", err);
             return;
           }
+
           setData(result);
         }
       );
-    }
+    };
 
     if (!store) {
       createStore();
     }
 
     return () => {
-      if (sub?.unsubscribe()) sub.unsubscribe();
-      if (store) store.close();
+      if (sub?.unsubscribe()) {
+        sub.unsubscribe();
```

Comment:
Is it wanted that `unsubscribe()` is called twice?

Reply:
It was in the original example, but I don't think it's wanted, so I'll remove the condition call 👍